core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4176,23 +4217,54 @@ private static void computeVersion(ServletContext context) {
     }
 
     /**
+     * The version number before it is "computed" (by a call to computeVersion()).
+     * @since FIXME
+     */
+    public static final String UNCOMPUTED_VERSION = "?";
+
+    /**
      * Version number of this Jenkins.
      */
-    public static String VERSION="?";
+    public static String VERSION = UNCOMPUTED_VERSION;
```

Comment:
It was `static` and `public` before this change (nothing to do with this PR). Is ref'd from other classes through the code.